qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/CassandraMetricsClientLimitMultiplier.java

```diff
@@ -31,16 +28,13 @@
 import com.palantir.remoting3.clients.ClientConfigurations;
 import com.palantir.remoting3.jaxrs.JaxRsClient;
 
-public final class CassandraMetricsClientLimitMultiplier implements ClientLimitMultiplier {
-    private final CassandraMetricsService cassandraMetricClient;
+public class CassandraMetricsClientLimitMultiplier implements ClientLimitMultiplier {
     private ThrottlingStrategy throttlingStrategy;
-    private Supplier<List<CassandraHealthMetric>> cassandraHealthMetrics;
+    private CassandraMetricMeasurementsLoader cassandraHealthMetrics;
```

Comment:
Should probably be final to be thread safe, since this will contain changing metric measurements.